Makefile

```
@@ -45,3 +46,7 @@ ci:
 .PHONY: docs
 docs:
 	$(MAKE) -C backend docs
+
+.PHONY: update-submodules
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)